.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+
+jobs:
+  build:
+    working_directory: ~/st2tests
+    parallelism: 1
+    shell: /bin/bash --login
+    docker:
+      - image: circleci/python:2.7
+    steps:
+      - checkout
+      - run: git clone -b master git@github.com:StackStorm/st2.git /tmp/st2
+      - run: make requirements
+      - save_cache:
+          key: v1-dep-{{ .Branch }}-{{ epoch }}
```

Comment:
This will create a new cache on every run. Seems a bit unnecessary, especially since we are never using that cache.

I think we should either:
1/ Not save the cache at all
2/ Save the cache using a key like `st2-v1-{{ checksum "requirements.txt" }}-{{ checksum "test-requirements.txt" }}`, **and** add an earlier `restore_cache` line

I think we are better

Reply:
Agreed.